rdfproxy/adapter.py

```diff
@@ -5,7 +5,6 @@
 
 from SPARQLWrapper import JSON, QueryResult, SPARQLWrapper
 from pydantic import BaseModel
-
```

Comment:
Please remove that removal, it has nothing to do with the actual change of the commit